apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/source/HealthCheckStreamCaller.java

```diff
@@ -0,0 +1,36 @@
+package gov.cms.bfd.pipeline.rda.grpc.source;
+
+import com.nava.health.v1.HealthCheckRequest;
+import com.nava.health.v1.HealthCheckResponse;
+import com.nava.health.v1.HealthGrpc;
+import gov.cms.bfd.pipeline.rda.grpc.PreAdjudicatedClaim;
+import io.grpc.ManagedChannel;
+import java.time.Duration;
+import java.util.Iterator;
+import java.util.concurrent.TimeUnit;
+
+/** GrpcStreamCaller implementation that calls the RDA HealthCheck service. */
+public class HealthCheckStreamCaller implements GrpcStreamCaller<HealthCheckResponse> {
```

Comment:
Are you thinking we can use this periodically in our pipeline job or is this just for testing out the RDA's gRPC service?

Reply:
This was to have a concrete gRPC implementation to code against to validate the framework in this sprint.  We certainly could use this code in the later sprints as well.